chapters/fa/chapter4/1.mdx

```diff
@@ -0,0 +1,23 @@
+<div dir="rtl">
+# هاب هاگینگ‌فیس
+
+[هاب هاگینگ‌فیس](https://huggingface.co/) –- وب‌سایت اصلی ما –- پلت‌فرمی مرکزی است که به همه امکان مشارکت، کشف و استفاده از جدیدترین و بهترین مدل‌ها و دیتاست‌ها را می‌دهد. این پایگاه، طیف گسترده‌ای از مدل‌ها را در خود جای داده، که بالغ بر ۱۰ هزار مورد از آن‌ها در دسترس عموم قرار دارد. در این فصل بر روی مدل‌ها متمرکز شده و در فصل ۵ نیز نگاهی به دیتاست‌ها خواهیم داشت.
+
+مدل‌های موجود در هاب، محدود به ترنسفورمرهای هاگینگ‌فیس یا حتی پردازش زبان طبیعی نیستند. از جمله آنها می‌توان مدل‌هایی از قبیل [Flair](https://github.com/flairNLP/flair) و [AllenNLP](https://github.com/allenai/allennlp) برای پردازش زبان طبیعی، [Asteroid](https://github.com/asteroid-team/asteroid) و [pyannote](https://github.com/pyannote/pyannote-audio) برای پردازش گفتار، و [timm](https://github.com/rwightman/pytorch-image-models) را برای پردازش تصویر برشمرد.
```

Comment:
> I previously mistook pardazesh-goftar for pardazesh-zaban-tabieei. Forgive me. I still think we should keep to the original acronym here tho. Please see the glossary and TRANSLATING/point2. This is an industry-accepted acronym.

I totally agree with your point here. Yet, I believe we've got to consider an exception for the second "NLP" occurrence on this page. I'm afraid a rendering view inconsistency arises when we use a Persian comma "،" right after an English word (being NLP here) in the whole sentence. See the attached screenshot.
Please let me know what you think about the right approach for the second one, I already modified the first occurrence to "NLP". 

![render view](https://user-images.githubusercontent.com/24595790/164526695-341b050e-bf40-40e8-a8db-5494f5b9a584.jpg)

Reply:
I think that might be an issue with your editor? But since the second sentence already has pardazeshe-goftar and pardazeshe-tasvir in it, I think your choice of pardazeshe-zaban-tabiei in the sentence totally works and sounds better. :)